app/views/layouts/nav/staff/_program_subnav.html.haml

```diff
@@ -0,0 +1,19 @@
+.navbar.navbar-default.program-subnav
+  .container-fluid
+    %ul.nav.navbar-nav
```

Comment:
Nice start.  Some minor changes for now.  We can worry about colors and styling later on.

Split nav bar into two sections (thinking 50/50 split to start):
  Left side includes program stats/info (leave empty for now)
  Right side includes nav links

Update nav items to the following in this order:
  Selection, gavel icon, ../program/selection, 'proposals#selection'
  Proposals, scale icon, ../program/proposals, 'proposals#index'
  Sessions, checkmark icon, ../program/sessions, 'programsessions#index'
  Speakers, people icon, ../program/speakers, 'speakers#index'

Note: I am include the ideal url for each link and where it will eventually link to.  For now fake or park wherever makes sense.